Mugcake.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Hallie's Recipes: About Me</title>
+    <link rel="stylesheet" href="index.css">
+</head>
+<body>
+<body>
+    <!-- this part is going to be the header of the website -->
+    <!-- this part is going to be the header of the website and is in every file -->
+    <div class = "header-flex">    
+        <a href = "C:\Users\halli\Hack4Impact\Milestone 1\bootcamp-project-2022\aboutme.html">About Me</a>
+        <a href = "C:\Users\halli\Hack4Impact\Milestone 1\bootcamp-project-2022\index.html">Home</a>
```

Comment:
Make sure these links are relative links instead of absolute links! Otherwise when run on other systems the links won't work for other systems that aren't `halli`.